library/src/main/java/com/novoda/downloadmanager/FilePath.java

```diff
@@ -1,5 +1,8 @@
 package com.novoda.downloadmanager;
 
+/**
+ * Represents a path on the system to the asset.
```

Comment:
same as before, it would be interesting to know if this returns a relative or an absolute path for example